F21/joelanzi/_toLowerCase.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+from unicodedata import normalize
+
+## To Lowercase Function
+def _to_lower(word, language):
+    lang = language[0:language.find('-')] if '-' in language else language
+    if lang == 'tr' or lang == 'az':
```

Comment:
Maybe adding a vector with all the exceptions like "tr", "az", "ga", etc., would permit an easier way to read all the exceptions instead of scrolling the script. To do this, you can add a line such as "if language is in **_vector_**, and then specify the code for each exception.